ports/thextech/README.MD

```diff
@@ -0,0 +1,79 @@
+## WHAT IS THEXTECH ?
+
+This is a direct continuation of the SMBX 1.3 engine. Originally it was written in VB6 for Windows, and later, it got ported/rewritten into C++ and became a cross-platform engine. 
+It completely reproduces the old SMBX 1.3 engine (aside from its Editor), includes many of its logical bugs (critical bugs that lead the game to crash or freeze got fixed), and also adds a lot of new updates and features.  
+This build is packed to be compatible with linux based handhelds that support Portmaster project (https://portmaster.games/)  
+Thanks to TheXTech project, Portmaster Community and @ds-sloth developer who supported me.
+
+## HOW TO BUILD
+
+For better compatibility TheXTech was build in Ubuntu 20.04 arm64 chroot.
+
+```
+git clone --recurse-submodules https://github.com/TheXTech/TheXTech.git
+```
+
+then edit CMakeLists.txt (or else it will not build inside such old environment) and add line:
+```
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
+```
+then we start to build:
+```
+mkdir build
+cd ./build
+cmake -DCMAKE_BUILD_TYPE=MinSizeRel -DUSE_SYSTEM_SDL2=ON -DTHEXTECH_BUILD_GL_DESKTOP_MODERN=OFF -DTHEXTECH_BUILD_GL_DESKTOP_LEGACY=OFF -DTHEXTECH_BUILD_GL_ES_LEGACY=OFF ..
+make
+```
+
+
+## HOW TO RUN
+
+Package contains only open source engine, for gaming you need assets (sprites, music and etc.), you can find them at the original github project https://github.com/TheXTech/TheXTech
+add assets into
+```
+./thextech/assets/assets_name_folder

```

Comment:
Create this directory with an empty text file named `./thextech/assets/assets_name_folder/.gitkeep` so that its easier for people to add assets inside.

Reply:
the thing is that assets name folder can be different , so name can be different

Reply:
Then maybe reword so it is more clear:

Package contains only open source engine, for gaming you need assets (sprites, music and etc.), you can find them at the original github project https://github.com/TheXTech/TheXTech add assets into `thextech/assets/<some_folder>`, for example:
- `thextech/assets/<something>/`

On first run it will do blah and create:
- `thextech/assets/<something else>/`

Reply:
I made changes in readme, I hope it is more understandable now

Reply:
Another change in Readme, now we agreed that it is ok in Discord!